dev.py

```diff
@@ -1035,6 +1035,11 @@ def task_meta(cls, list_targets, parallel, args, **kwargs):
         if parallel:
             make_params.append(f'SPHINXOPTS="-j{parallel}"')
 
+        # Environment variables needed for notebooks
+        # See gh-17322
+        make_params.append('SQLALCHEMY_SILENCE_UBER_WARNING=1')
+        make_params.append('JUPYTER_PLATFORM_DIRS=1')
```

Comment:
I would be fine with these.

(In the same spirit of the dep tracker issue, maybe we should have a place to note things we need to do when something updates. Be it a Python version or another library. Otherwise this might stay there longer than intended, not that it would be a big issue though.)